tweets/async-hooks.tweet

```diff
@@ -0,0 +1 @@
+If you use async_hooks with #nodejs please make sure to read https://nodejs.medium.com/what-use-cases-exist-for-async-hooks-6eab74d7eefd and comment on the issue so that we have your use case in the list.
```

Comment:
I moved the link to the end of the tweet so Twitter will display it best and I tweaked the language a little bit.

```suggestion
If you use async_hooks with #nodejs please read our recent blog post and comment on the Github issue (linked in blog post) so that we have your use case in the list. Thanks!

https://nodejs.medium.com/what-use-cases-exist-for-async-hooks-6eab74d7eefd
```